package-files/postinstall.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+HOMEDIR='/etc/infra'
+
+set -eu
+
+if command -v systemctl >/dev/null 2>&1; then
+  systemctl enable infra
+fi
```

Comment:
I don't think we should enable the `infra` service. This is going to make it impossible to re-use this package for installing the CLI or the server.

Let's make the package only install things, and let the user start the services once they are correctly configured. That's how all arch linux packages work, and I think it's a much better experience than I've seen from packages in other distros.